src/grid-list/grid-list.jsx

```diff
@@ -56,6 +56,7 @@ const GridList = React.createClass({
       root: {
         display: '-webkit-box; display: -moz-box; display: -ms-flexbox; display: -webkit-flex; display: flex',
         flexWrap: 'wrap',
+        WebkitFlexWrap: 'wrap',
```

Comment:
We are using https://github.com/rofrischmann/inline-style-prefixer so we shouldn't need it.
I think that we should remove the prefixes from the obove `display` property